test/e2e/suite/conformance/certificates/venafi/venafi.go

```diff
@@ -19,32 +19,32 @@ package venafi
 import (
 	"context"
 
-	"github.com/jetstack/cert-manager/test/e2e/suite/conformance/certificates"
-
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
 	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
 	"github.com/jetstack/cert-manager/test/e2e/framework"
+	"github.com/jetstack/cert-manager/test/e2e/framework/helper/featureset"
 	"github.com/jetstack/cert-manager/test/e2e/framework/util/errors"
+	"github.com/jetstack/cert-manager/test/e2e/suite/conformance/certificates"
 	vaddon "github.com/jetstack/cert-manager/test/e2e/suite/issuers/venafi/addon"
 )
 
 var _ = framework.ConformanceDescribe("Certificates", func() {
 	// unsupportedFeatures is a list of features that are not supported by the
 	// Venafi issuer.
-	var unsupportedFeatures = certificates.NewFeatureSet(
-		certificates.DurationFeature,
+	var unsupportedFeatures = featureset.NewFeatureSet(
+		featureset.DurationFeature,
 		// Due to the current configuration of the test environment, it does not
 		// support signing certificates that pair with an elliptic curve private
-		// key or using the same private key multiple times.
-		certificates.ECDSAFeature,
-		certificates.EmailSANsFeature,
-		certificates.URISANsFeature,
-		certificates.IPAddressFeature,
-		certificates.OnlySAN,
+		// key
+		featureset.ECDSAFeature,
+		featureset.EmailSANsFeature,
+		featureset.URISANsFeature,
+		featureset.IPAddressFeature,
+		featureset.OnlySAN,
```

Comment:
Add some comments explaining why these other features are not supported on Venafi Cloud and / or TPP.
I think you can configure  TPP to allow  URISANs, it's buried in aperture web UI, IIRC.

Reply:
the TPP ones were added by James ages ago... think back then we didn't know this option... would be a good follow up PR?